tests/test_auth.py

```diff
@@ -33,3 +33,46 @@ def test_on_prem_url():
     assert user1 == 'user1'
     assert user2 == 'user2'
     assert user3 == 'user3'
+
+from conftest import USR_BASE_URL, USERNAME, DEFAULT_PUBLIC_API_KEY
+
+def test_api_key_auth_client_username():
+    conf_username = APIKeyAuthClient(USR_BASE_URL, API_KEY).username
+    assert conf_username == USERNAME
+
+def test_api_key_auth_client_me_endpoint():
+    client = APIKeyAuthClient(USR_BASE_URL, API_KEY)
+    username = client.send('/api/v3/me', 'get').json()['config']['user_name']
+    assert username == USERNAME
+
+from carto.auth import AuthAPIClient
```

Comment:
According to PEP8, imports should be at the top: http://legacy.python.org/dev/peps/pep-0008/#imports

There are linters based on pep8, which you may find interesting. Also, what about adding Hound to this project? :trollface:

Reply:
Ok, file changed to be pep8-compilant. I've enabled pep8 reviews at IntelliJ, I'm just not used to them yet :_)